### PR TITLE
Updated Profile

### PR DESCRIPTION
**Pull request by mattscott68**
Updated profile with photo for Matthew Scott Tan

### CODE REVIEW

File: _members/matthew-scott-tan.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+services:
+  #home: https://member-website.com
+  #github: https://github.com/
+  #twitter: https://twitter.com/
+  #linked-in: https://www.linkedin.com/in/
+
+start: Sep 2023
+end:
+---
+Matthew is an udergraduate stuent concentrating in Mathematics and Statistics at Harvard College. Previously, he was a Specialist (Intelligence) in the Singapore Armed Forces, focusing on imagery and geospatial analysis. At the Gehlenborg lab, he is using computational and mathematical methods to process single-cell resolution omics and imaging data.
```
Comment (NickAkhmetov) [inline _members/matthew-scott-tan.md:19]: Minor spelling fix + I believe we've changed the lab name to HIDIVE:

```suggestion
Matthew is an undergraduate student concentrating in Mathematics and Statistics at Harvard College. Previously, he was a Specialist (Intelligence) in the Singapore Armed Forces, focusing on imagery and geospatial analysis. At the HIDIVE lab, he is using computational and mathematical methods to process single-cell resolution omics and imaging data.
```

File: _members/matthew-scott-tan.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+job_title: Undergraduate Student in Mathematics and Statistics
+role: student # or postdoc, administration, staff, alumni, etc.
+
+services:
+  #home: https://member-website.com
+  #github: https://github.com/
+  #twitter: https://twitter.com/
+  #linked-in: https://www.linkedin.com/in/
+
+start: Sep 2023
```
Comment (NickAkhmetov) [inline _members/matthew-scott-tan.md:16]: Other members' start dates have used the full month name:

```suggestion
start: September 2023
```

**Review by NickAkhmetov — COMMENTED** [review]
A few minor fixes then it should be good to go!

**Comment by mattscott68** [comment]
Awesome, thanks a lot!

File: _members/matthew-scott-tan.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+name_degree: Matthew Scott Tan
+
+photo: matthew-scott-tan.png
+
+job_title: Undergraduate Student in Mathematics and Statistics
+role: student # or postdoc, administration, staff, alumni, etc.
+
+services:
+  #home: https://member-website.com
+  #github: https://github.com/
```
Comment (lchoy) [inline _members/matthew-scott-tan.md:12]: Would you like to include your GitHub profile?

```suggestion
  github: https://github.com/mattscott68
```

File: _members/matthew-scott-tan.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+  #twitter: https://twitter.com/
+  #linked-in: https://www.linkedin.com/in/
```
Comment (lchoy) [inline _members/matthew-scott-tan.md:14]: or LinkedIn profile?

```suggestion
  linked-in: https://www.linkedin.com/in/matthew-scott-tan/
```

**Review by lchoy — APPROVED** [review]

**Pull request merged**